config/jobs/kubernetes/test-infra/test-infra-periodics.yaml

```diff
@@ -19,31 +19,21 @@ periodics:
       image: busybox
       args: ["cat", "config/jobs/kubernetes/test-infra/test-infra-periodics.yaml"]
 - name: ci-test-infra-bazel
+  decorate: true
   interval: 1h
   labels:
     preset-service-account: "true"
     preset-bazel-scratch-dir: "true"
-    preset-bazel-remote-cache-enabled: "true"
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190329-811f7954b-experimental
+    - image: launcher.gcr.io/google/bazel:0.24.1
+      command:
+      - bazel
       args:
-      - "--job=$(JOB_NAME)"
-      - "--repo=k8s.io/test-infra=master"
```

Comment:
you'll need an extra_ref for test-infra I think

Reply:
Done